tests/test_functional.py

```diff
@@ -32,8 +32,16 @@ def test_no_t3_no_qm():
                               'termination_time': 1e-3}],
                 'model': {'core_tolerance': 0.01}}
 
+    t3_args = {'sensitivity':
```

Comment:
This is a "no_t3" functional test. Do we have to make this addition?

Reply:
With the previous behavior, the functional test failed without it. With a minor modification to writer, this t3 block is no longer necessary.